frontend/src/components/Diff/Diff.tsx

```diff
@@ -21,13 +21,13 @@ const SelectedSourceLineContext = createContext<number | null>(null)
 
 function FormatDiffText({ texts }: { texts: api.DiffText[] }) {
     return <> {
-        texts.map(t => {
+        texts.map((t, i) => {
```

Comment:
[This is a poor fix](https://robinpokorny.medium.com/index-as-a-key-is-an-anti-pattern-e0349aece318) - I'd rather keep the warning rather than use the index as the key.
I'm not really sure what a suitable key _would_ be, though

Reply:
I think this falls under the "static content" exception mentioned in that post:

> 1.    the list and items are static–they are not computed and do not change;
> 2.    the items in the list have no ids;
> 3.    the list is never reordered or filtered.
>
> When all of them are met, you may safely use the index as a key.

We shouldn't be updating individual lines in-place here, so it's fine to silence the warning.

Reply:
That's true for now, my only worry would be with future #115 changes. But I guess this is alright for now, ty!